store/store_bench_test.go

```diff
@@ -194,7 +194,7 @@ func benchStoreSet(b *testing.B, valueSize int, process func(interface{}) ([]byt
 		}
 	}
 
-	kvs = nil
```

Comment:
Think the tool is wrong. It assumes `kvs = nil` is a new assignment, so `kvs` needs to be used again.